x-pack/packages/kbn-elastic-assistant/impl/assistant/chat_send/use_chat_send.tsx

```diff
@@ -136,6 +139,9 @@ export const useChatSend = ({
       assistantTelemetry?.reportAssistantMessageSent({
         conversationId: currentConversation.title,
         role: responseMessage.role,
+        actionTypeId: currentConversation.apiConfig.actionTypeId,
+        model: currentConversation.apiConfig.model,
+        provider: currentConversation.apiConfig.provider,
```

Comment:
Just want to confirm, but we won't get `model` and `provider` for `pre-configured` connectors, correct? That's what I'm seeing in testing at least.

Reply:
correct, I think the lack of the field + the presence of actionTypeId will imply preconfigured. I think that should be enough to make a Lens viz from?

Reply:
Yeah, that seems fine. Can always make a runtime field instead of calculating before and including as an explicit field.